addon/services/power-calendar.js

```diff
@@ -10,6 +10,9 @@ export default class extends Service {
   get locale() {
     return getDefaultLocale();
   }
+  set locale(value) {
+    return this._locale = value
```

Comment:
I don't think you need the `this._locale = value`, just `return value`

Reply:
good call. updated!